common/app/views/fragments/inlineJSBlocking.scala.html

```diff
@@ -30,6 +31,9 @@
     // page config
     @Html(config(page).body)
 
+    // permutive
+    @InlineJs(permutive().body, "permutive.js")
```

Comment:
As `permutive`is a third party tag, I think he should be in `third-party-tags.js` like `krux` 
I don't think we can load it without first retrieve the consent from the reader.

Reply:
@mchv there are two parts to the permutive implementation, on is the library (which I'm loading from third party tags) but the head snippet is obligatory for permutive to work and it needs to run before any call to gtag.enableServices(). The snippet on it's own however doesn't do anything if the library isn't loaded, which I'm loading on user consent only. 
I have written up a doc this morning about the implementation which I will share with you.

Reply:
I caught up with @faresite and he explained me how this part do not load any code from a third-party, only needed configuration. So that specific point looks good to me 👍